handbook/FileService-MinIO/FileService-MinIO.md

```diff
@@ -0,0 +1,59 @@
+# Leverage Cloud File Storage with MinIO
+
+SystemLink File Service allows you to configure and use the Amazon S3 cloud storage instead of a file share. While NI officially addresses the Amazon S3 cloud storage, the File Service also works with MinIO, a server-side software storage stack that is compatible with Amazon S3.
+
+To use MinIO with SystemLink, you need SystemLink version 2020 R3 or later.
+
+To use MinIO as a storage provider, set up the MinIO server on a system that you would like to upload files to. This can be the same machine as the SystemLink server or an own dedicated server. Then, configure the File Service to use that server.
+
+## Running MinIO server
+
+1. Download the server application from the [MinIO website](https://min.io/download).
+
+2. From the command line, run the following command to run the server:
+
+    ```bash
+    minio server <PathToRootFolder>
+    ```
+
+    <figure>
+    <img src="../../img/minio-server.png" width="500" />
+    <figcaption>Running a minio server.</figcaption>
+    </figure>
+
+3. When starting the MinIO server for the first time, use a web browser to connect to the endpoint that the command line prints out. Create a bucket by using the `+` button in the right bottom corner.
+
+    !!! note "Note"
+        - It is recommended to set non-default access key and secret key. You can do that by setting the environment variables `MINIO_ACCESS_KEY` and `MINIO_SECRET_KEY`. Run the following two commands from the command line just before starting the server.
+        ```
+        set MINIO_ACCESS_KEY=YourAccessKey
+        set MINIO_SECRET_KEY=YourSecretKey
+        ```
+        - There's no bucket region necessary for using MinIO, but you have to apply a bucket region for the File Service to work. Just use `us-east-1`.
```

Comment:
Does `us-east-1` apply for all global customers? I know you said it's not necessary but maybe we could provide a bit more detail here.

Reply:
It sounds like you just need a string here that matches the SL configuration. We should probably make that clear, and that we are using 'us-east-1' in examples for convenience.

Reply:
'us-east-1' is just an arbitrary but valid bucket region. It will work for any customer because minio is not hosted on AWS. I added this reasoning to clarify. I also added a second example just to decouple a bit more from the expectation that it had to be us-east-1.